.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set common environment variables
+        run: |
+          echo "build_date=$(date -u '+%Y%m%dT%H%M%SZ')" >> $GITHUB_ENV
+          echo "build_version=$(cat VERSION)" >> $GITHUB_ENV
+
+      - name: Build Docker base image
+        run: |
+          docker pull digitalmarketplace/base
+          docker build --pull --cache-from digitalmarketplace/base --build-arg BUILD_DATE=${{ env.build_date }} --build-arg BUILD_VERSION=${{ env.build_version }} -t digitalmarketplace/base -f base.docker .
+
+      - name: Build API image
+        run: docker build -f api.docker .
```

Comment:
Nit: would make it easier to review/understand with the long version of options (`--file`)

Reply:
Good point, added in bd13de8